app.js

```diff
@@ -0,0 +1,98 @@
+
+
+//array to hold all the pctures as objects
+
+Picture.allPictures = [];
+
+//make a constuctor function for picture objects
+
+function Picture(filepath, name) {
+  this.filepath = filepath;
+  this.name = name;
+  this.totalClicks = 0;
+  Picture.allPictures.push(this);
+}
+
+//make the objects
+
+new Picture('img/bag.jpg', 'Bag');
+new Picture('img/banana.jpg', 'Banana cutter');
+new Picture('img/bathroom.jpg', 'TP holder with tablet mount');
+new Picture('img/boots.jpg', 'Open toe rubber boots');
+new Picture('img/breakfast.jpg', 'Breakfast maker');
+new Picture('img/bubblegum.jpg', 'Meatball bubble gum');
+new Picture('img/chair.jpg', 'Bumpy chair');
+new Picture('img/cthulhu.jpg', 'Cthulhu figurine');
+new Picture('img/dog-duck.jpg', 'Duck lips for dogs');
+new Picture('img/dragon.jpg', 'Canned dragon meat');
+new Picture('img/pen.jpg', 'Pen with cuttlery heads');
+new Picture('img/pet-sweep.jpg', 'Doog boot-mops');
+new Picture('img/scissors.jpg', 'Pizza cutting scissors');
+new Picture('img/shark.jpg', 'Shark sleeping bag');
+new Picture('img/tauntaun.jpg', 'Tauntaun sleeping bag');
+new Picture('img/unicorn.jpg', 'Canned unicorn meat');
+new Picture('img/usb.gif', 'Tentacle usb drive');
+new Picture('img/water-can.jpg', 'Watering can');
+new Picture('img/wine-glass.jpg', 'Wineglass');
+
+console.log(Picture.allPictures);
+
+//access the element
+
+var picElement = document.getElementById('pic1');
+var picElement2 = document.getElementById('pic2');
+var picElement3 = document.getElementById('pic3');
+
+
+// event listener
+
+picElement.addEventListener('click', giveThreePics);
+picElement2.addEventListener('click', giveThreePics);
+picElement3.addEventListener('click', giveThreePics);
+
+
+//function to count
+
+function clickCounter(event) {
+
```

Comment:
Need this to detect when I've made 25 votes and then disable the event listener and write the report.